src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -289,8 +292,30 @@ export class PiaIntakeService {
       invitee: {
         createdByGuid: user.idir_user_guid,
       },
+      status: Not(PiaIntakeStatusEnum.COMPLETE),
     });
 
+    // Scenario 5: Return COMPLETE PIAs only if queried and if the user is a drafter, invitee, or MPO on the PIA
+    if (query.filterByStatus === PiaIntakeStatusEnum.COMPLETE) {
+      whereClause.push({
+        ...commonWhereClause,
+        createdByGuid: user.idir_user_guid,
+        status: PiaIntakeStatusEnum.COMPLETE,
+      });
+      whereClause.push({
+        ...commonWhereClause,
+        invitee: {
+          createdByGuid: user.idir_user_guid,
+        },
+        status: PiaIntakeStatusEnum.COMPLETE,
+      });
+      whereClause.push({
+        ...commonWhereClause,
+        ministry: In(mpoMinistries),
+        status: PiaIntakeStatusEnum.COMPLETE,
+      });
+    }
+
```

Comment:
Scenario 5 to override may not be required at all.

We already have the code that adds the specific status down at line 317/342 (search **filter logic here** on the file)